lib/pack.js

```diff
@@ -163,6 +163,7 @@ function packGitDep (manifest, dir) {
       const child = cp.spawn(process.env.NODE || process.execPath, [
         require.main.filename,
         'install',
+        '--only=dev',
```

Comment:
This will only install `devDependencies` and not `dependencies`. Did you want to use `--dev`?